.github/workflows/ktlint.yml

```
@@ -0,0 +1,42 @@
+
+permissions:
+    contents: read
+    pull-requests: write
+    checks: write
+
+jobs:
+    ktlint:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   name: Checkout
```

YAMLlint (1.38.0)

[error] too many spaces after hyphen

(hyphens)